code/src/Author.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+export const Author = ( { favAuthor, onFavouriteAuthorChange }) => {
+    return (
+        <>
+            <section className="question">
+                <label htmlFor="author" className="text-question">
+                    Who is your favourite author?
+                    <input
+                        id="author"
+                        className="text-input"
+                        type="text"
+                        placeholder="Type..."
+                        onChange={event => onFavouriteAuthorChange(event.target.value)}
+                        value={favAuthor}
+                        required
+                    />
+                </label>
```

Comment:
Good that you have Label and also htmlFor and id! 
Fun with the placeholder and good indentation, this is very easy to read!